htdocs/api/v0.0.1/.htaccess

```diff
@@ -0,0 +1,25 @@
+Header always set Access-Control-Allow-Origin "*"
+Header always set Access-Control-Allow-Headers "*"
+RewriteEngine on
+Options -Indexes
+
+
+# Projects API rewrite rules
+RewriteRule ^projects/([a-zA-Z0-9_\w\s.]+)/instruments/([a-zA-Z0-9_.]+)$ projects/InstrumentForm.php?Instrument=$2&PrintInstrumentForm=true [END]
```

Comment:
The RewriteRule flag 'END' was introduced in apache v2.4. It seems that most of are LORIS vms are running on version 2.2. According to the apache documentation (http://httpd.apache.org/docs/2.4/rewrite/flags.html) the 'END' tag does everything like th 'L' tag  but also prevents any subsequent rewrite processing from occurring in per-directory (htaccess) context.